tests/data_protection/oadp/utils.py

```diff
@@ -174,6 +174,7 @@ def create_rhel_vm(
 
 
 def check_file_in_vm(vm):
+    wait_for_running_vm(vm=vm)
```

Comment:
logically, wait_for_running_vm should be a part of "restore backup" step

when you restored the backup you should check if the VM is ok and running and only after that move to next step